test/test_tensor_creation_ops.py

```diff
@@ -1145,6 +1145,12 @@ def test_full_out(self, device):
         self.assertEqual(torch.full(o.shape, 1., out=o).dtype, o.dtype)
         self.assertEqual(torch.full(size, 1, out=o).dtype, o.dtype)
 
+    def test_tensor_from_writable_numpy(self, device):
```

Comment:
if I understand correctly shouldn't this be 
```suggestion
    def test_tensor_from_non_writable_numpy(self, device):
```